UiPath/README.md

```diff
@@ -21,3 +21,27 @@ Select CradlAI.Activities.Designer in the project menu and press _Build_ in the
 
 # Testing in UIPath Studio
 Open _Manage Packages_ -> Settings and add the path to your local build. You should now be able to find the local package and test it in your flow.
+
+
+# Listing information
+
+## Title
+Cradl AI - Automated Data Extraction Demo
+
+## Asset Pitch 
+Boost UiPath with Crad AI: State of the art document parsing powered by the latest within machine learning.
+
+## Features
+Two illustrative examples of how to use Cradl AI in UiPath
+1. Send your document to Cradl AI for data extraction and optional human verification. 
+2. Retrieve extracted data from Cradl AI.
+A feedback loop is automatically created so your model gets better for every document you parse with Cradl AI.
+
+## About Us
+
+Build cutting edge AI models for document parsing and seamlessly integrate into your existing workflows in UiPath. 
+Instead of relying on pre-trained models that almost fit your case, Cradl AI enables you to train your own model on your own documents. 
```

Comment:
```suggestion
Instead of relying on pre-trained models that almost fit your case, Cradl AI enables you to tailor your model to your own documents. 
```